cmd/bootstrap/cmd/rootblock.go

```diff
@@ -144,7 +152,14 @@ func rootBlock(cmd *cobra.Command, args []string) {
 
 	chainID := parseChainID(flagRootChain)
 	if (chainID == flow.Testnet || chainID == flow.Mainnet) && flagRootView == 0 {
-		log.Fatal().Msgf("--root-view must be non-zero for %q chain", flagRootChain)
+		log.Fatal().Msgf("--root-view must be non-zero on %q chain", flagRootChain)
+	}
+
+	// Warn if using default values on mainnet/testnet
+	if (chainID == flow.Testnet || chainID == flow.Mainnet) &&
+		flagFinalizationSafetyThreshold == defaultFinalizationSafetyThreshold &&
```

Comment:
this looks like a workaround, I assume flags package have means to check if the value was set.